deploy/crds/alerts_v1alpha1_storagealert_cr.yaml

```diff
@@ -1,11 +1,48 @@
 apiVersion: alerts.monstorak.org/v1alpha1
 kind: StorageAlert
```

Comment:
Why are we creating 3 cr ? for two monitoring projects?  why can't we have single(https://github.com/monstorak/monstorak/pull/32/files#diff-b81715432dc97f582ba99e9d4dadc3a7R31) cr which hold all the monitoring

Reply:
> I agree with @cloudbehl here. We can have something like below
> 
> ```
> apiVersion: alerts.monstorak.org/v1alpha1
> kind: StorageAlert
> metadata:
>   name: ceph-storagealert
>   namespace: storage-monitoring
> spec:
>   storage:
>   - provider: "ceph"
>     version:
>     - "v14.2.1-20190430"
>     namespace: "openshift-storage"
>   - provider: "noobaa"
>     version:
>     - "xxxxxx"
>     namespace: "openshift-storage"
>   prometheus:
>     namespace: "openshift-monitoring"
>     label: "prometheus:k8s"
> ```

That is exactly what we have.

Reply:
> Why are we creating 3 cr ? for two monitoring projects? why can't we have single(https://github.com/monstorak/monstorak/pull/32/files#diff-b81715432dc97f582ba99e9d4dadc3a7R31) cr which hold all the monitoring

If you look closely, these are different ways to create a CR.
You can create as many CRs as required. These are the ways you can do it.

Reply:
Ok.